api/nodes/serializers.py

```diff
@@ -124,6 +127,123 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NodeContributorsSerializer(JSONAPISerializer):
+    """ Separate from UserSerializer due to necessity to override almost every field as read only
+    """
+    filterable_fields = frozenset([
+        'fullname',
+        'given_name',
+        'middle_name',
+        'family_name',
+        'id',
+        'bibliographic',
+        'permissions'
+    ])
+    id = ser.CharField(source='_id')
+    fullname = ser.CharField(read_only=True, help_text='Display name used in the general user interface')
+    given_name = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    middle_name = ser.CharField(read_only=True, source='middle_names', help_text='For bibliographic citations')
+    family_name = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    suffix = ser.CharField(read_only=True, help_text='For bibliographic citations')
+    date_registered = ser.DateTimeField(read_only=True)
+    gravatar_url = ser.CharField(read_only=True, help_text='URL for the icon used to identify the user. '
+                                                           'Relies on http://gravatar.com ')
+    social_accounts = ser.DictField(read_only=True, source='social',
+                                    help_text='A dictionary of various social media account dentifiers including '
+                                              'an array of user-defined URLs')
+    bibliographic = ser.BooleanField(help_text='Whether the user will be included in citations for this node or not.  '
+                                               'Required due to issues with field defaulting to false.')
+
+    permission = ser.ChoiceField(choices=['read', 'write', 'admin'], required=False, allow_null=True,
```

Comment:
For consistency (and to clarify that this ties to the permissions system), perhaps it would be better to use the named constants `osf_permissions.READ`, `osf_permissions.WRITE`, and `osf_permissions.ADMIN`?

(These names won't change much, so this is minor, but nice to have)